arviz/utils.py

```diff
@@ -80,3 +81,19 @@ def wrapper(function):
         return wrapper(function)
     else:
         return wrapper
+
+
+def wrapper(function, *args, **kwargs):
+    def wrapped():
+        return function(*args, **kwargs)
+
+    return wrapped
+
+
+def to_numba_or_not_to_numba(function, *args, **kwargs):
```

Comment:
I understand the idea of this function if for us to manually check if a adding numba is helping or not. Right? I still think we should also have tests in order to detect regressions (not only from our side, but just in case numba changes in some unexpected way). @ColCarroll what do you think about this?

Reply:
How accurate timing is on travis. 

Can we send those timings to some server so we can plot against each time they are tested?

Reply:
Yes, this gives us an estimate of the numba enhancement. I am thinking of adding a loop in this method so as to get a proper estimate of the time. If everyone is fine with this particular method of testing numba then I'll add the test.

Reply:
We use [airspeed velocity](https://github.com/airspeed-velocity/asv) on pymc3. The benchmarks are here: http://pandas.pydata.org/speed/pymc3/, and we check them manually whenever we want to. 

One of the anaconda devs (and all around good person) helped us set that up - I could check whether we can also run `arviz` benchmarks on that machine.